### PR TITLE
Update_happy-new-year.sh

### PR DESCRIPTION
**Pull request by arnabmukho**

### CODE REVIEW

**Comment by tachyons** [comment]
🥇

**Comment by balasankarc** [comment]
+1

**Comment by vivekkj123** [comment]
LGTM 👀

**Comment by subins2000** [comment]
@arnabmukho Hi, what is the point of this PR? Why change the name?

File: happy-new-year.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Config
-name='subins2000'
+name='Arnab :)'
```
Comment (tachyons) [inline happy-new-year.sh:4]:
```suggestion
name='Arnab 😀'
```

**Review by tachyons — CHANGES_REQUESTED** [review]

**Pull request closed without merging**